_posts/2021-05-14-bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton.md

```diff
@@ -0,0 +1,612 @@
+You can visit _localhost:8443_ in order to connect Argo CD UI's screen but for Tekton, you need to install _Tekton Dashboard_.
+
+### 5. Install Tekton Dashboard
+
+Tekton has another great project called [Tekton Dashboard](https://github.com/tektoncd/dashboard). Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines and Tekton triggers resources. We can easily install this to our cluster and see what's goin' on our cluster. Run the following command to install Tekton Dashboard and its dependencies:
+
+```bash
+$ kubectl apply --filename https://storage.googleapis.com/tekton-releases/dashboard/latest/tekton-dashboard-release.yaml
+customresourcedefinition.apiextensions.k8s.io/extensions.dashboard.tekton.dev created
+serviceaccount/tekton-dashboard created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-backend created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-dashboard created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-extensions created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-pipelines created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-tenant created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-triggers created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-backend created
+service/tekton-dashboard created
+deployment.apps/tekton-dashboard created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-pipelines created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-dashboard created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-triggers created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-tenant created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-extensions created
+```
+
+You can simple access to your dashboard with running the following command:
+
+```bash
+$ kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097
+Forwarding from 127.0.0.1:9097 -> 9097
+Forwarding from [::1]:9097 -> 9097
+
+$ open http://localhost:9097
+```
+
+I recommend you to divide screen into two part, one for Argo CD UI and one for Tekton UI to follow the process before making any changes on your repository.
+
+![follow_process](/images/bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton/follow_process.png)
+
+### 6. Test
+
+Finally, let's test our function, to do so, we should access the OpenFaaS Gateway component.
+
+```bash
+$ kubectl port-forward svc/gateway -n openfaas 8081:8080
+Forwarding from 127.0.0.1:8081 -> 8080
+Forwarding from [::1]:8081 -> 8080
+
+$ httpie POST http://localhost:8081/function/hellofunction.default message="Hello World"
+HTTP/1.1 200 OK
+Content-Length: 35
+Content-Type: text/plain; charset=utf-8
+Date: Fri, 07 May 2021 08:35:44 GMT
+X-Call-Id: 0ee19dd7-2c09-4093-9b6e-0755836afb9c
+X-Duration-Seconds: 0.004628
+X-Start-Time: 1620376544209429100
+
+Body v6: {"message": "Hello World"}
+```
+
+Tadaaaa 🎉😋✅
```

Comment:
This was a really long blog post and we did so much, so please summarise it all here instead of finishing with just Tada :slightly_smiling_face: